init_unix.go

```diff
@@ -5,7 +5,11 @@ package main
 import "os"
 
 func init() {
```

Comment:
How about the following implementation? It is matter of preference so, you do not necessarily have to apply it.

```go
paths := []string{"/sbin", "/usr/sbin", "/usr/bin"}
if p := os.Getenv("PATH"); p != "" {
   paths = append(paths, p)
}
os.Setenv("PATH", strings.Join(paths, string(filepath.ListSeparator)))
```